src/mcts/search.cc

```diff
@@ -644,7 +644,7 @@ void Search::Wait() {
 }
 
 Search::~Search() {
-  Abort();
+  if (!stop_.load(std::memory_order_acquire)) Abort();
```

Comment:
After some thought, I think it's even better to do that in Abort(). If `stop_` is already true, just exit (maybe also output log message).
That way if Stop() was called before Abort(), stop wins. Otherwise Abort() wins.

But we don't have situations like this in the code now, so current way is totally fine too.

Reply:
I'm not sure which version I prefer.